src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -111,6 +112,18 @@ void set(final Event e, final String message) {
 					((PlayerQuitEvent) e).setQuitMessage(message);
 			}
 		},
+		UNKNOWN("unknown", "[the] [event-]unknown command [message]", UnknownCommandEvent.class) {
```

Comment:
We are referring to the message not the command itself so this must be compulsory. `[the]` is already added in the constructor below. `[event-]` part shouldn't be used here.
```suggestion
		UNKNOWN("unknown", "unknown command( |-)message", UnknownCommandEvent.class) {
```